src/lib/CRSF/CRSF.cpp

```diff
@@ -32,7 +30,7 @@ HardwareSerial CRSF::Port = Serial;
 GENERIC_CRC8 crsf_crc(CRSF_CRC_POLY);
 
 /// Out FIFO to buffer messages///
-static FIFO SerialOutFIFO;
+static FIFO<256> SerialOutFIFO;
```

Comment:
256 seems to be used in >1 places in this file.... should it be turned into a `#define`?
Also... DEATH TO MAGIC NUMBERS!

Reply:
Added constants for magic numbers 💀